src/components/__tests__/AgencyList.test.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { AgencyItem } from "../UI/AgencyList/AgencyList";
+import agencyData from "../../assets/data/agencyData";
+import "@testing-library/jest-dom/extend-expect";
+
+describe("AgencyItem Component", () => {
+  const item = agencyData[0];
+
+  test("AgencyItem displays item details", () => {
+    render(<AgencyItem item={item} />);
+    expect(screen.getByText(item.title)).toBeInTheDocument();
```

Comment:
Here you can check that the AgencyItem has correct text content:
```
    expect(screen.getByText(item.title)).toHaveTextContent('Göteborg South');
```

Reply:
Same for other checks in this test

Reply:
Do I do it for other agencies?